domain/src/main/java/com/example/domain/item.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.domain
+
+data class Item(
+    val header: Header,
+    var body: Body
+) {
+    fun priceString(price: Float): String {
+        return price.toString()
+    }
+}
+
+data class Header(
+    val code: Int,
+    val message: String?
+)
+
+data class Body(
+    var data: Data
+)
+
+data class Data(
+    var base: List<Base>
```

Comment:
For `Body.data` or `Data.base`, why do you declare this member variables as an immutable variable? is there any reason?